content/en/users/_index.md

```diff
@@ -9,75 +10,17 @@ menu:
     pre: <i class='fa fa-user-friends'></i>
 ---
 
-## Request for information
-
-You can ask for more information about the services
-[here](https://www.egi.eu/more-information/).
-
-## Accessing EGI services
-
-You can access EGI services from:
-
-- [EGI Service Catalogue](https://www.egi.eu/services/)
-
-Depending on the access conditions, a service (or an instance of the service)
-may be open for any user, OR it may require requesting access (Ordering). The
-EGI Website together with the connected EGI Marketplace streamlines the process.
-
-## Access conditions
-
-EGI services use the following types of access conditions:
+This section offers an [introduction](getting-started) to the
+EGI Federated Cloud (FedCloud) and contains detailed documentation for
+[public EGI services](https://www.egi.eu/services/), togheter with
+[tutorials](tutorials) about how to set up, use, and combine these services.
 
-- Wide access: Users can freely access the service. Login may be required but
-  it's possible with various institutional accounts (through EduGAIN) or with a
-  social IDs (e.g. Google). Example: the
-  [open instance of the EGI Notebooks](https://notebooks.egi.eu/)
-- Policy based: Users are granted access based on specific policies defined by
-  the service providers. Access needs to be requested, and will be checked for
-  such services. Example: Compute resources and tools allocated to researchers
-  in medical imaging ([Biomed VO](http://lsgc.org/biomed.html)).
-- Pay-for-use: Services are provided for a fee. Example:
-  [FitSM In-house course](https://www.egi.eu/services/fitsm-training/in-house-training/)
+{{% alert title="Note" color="info" %}} See the [Service Providers](../providers)
+section for details on how to integrate identity providers and service providers
```

Comment:
I would just say "how to integrate providers into EGI Federation"

Reply:
Fixed. Will have to update section Internal Services too, as there I used the same wording.